src/Service.Host/Service.Host.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.5.002.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Service.Host", "Service.Host.csproj", "{D932E30C-B821-44D2-A0B6-6F1B01100A57}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{D932E30C-B821-44D2-A0B6-6F1B01100A57}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{D932E30C-B821-44D2-A0B6-6F1B01100A57}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{D932E30C-B821-44D2-A0B6-6F1B01100A57}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{D932E30C-B821-44D2-A0B6-6F1B01100A57}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {66CC0062-3269-41A4-A8AF-699FFFE50CA8}
+	EndGlobalSection
```

Comment:
visual studio keeps adding this extra .sln file in that causes the docker buiild stage to fail. Not sure why. I thought I had added this to the .gitignore already but evidently not